src/constants/types.tsx

```diff
@@ -148,76 +149,12 @@ export type CustomerT = {
   mobile?: string;
 };
 
-export type PaletteItemT = {
```

Comment:
This is probably an issue from solving conflicts, but we shouldn't remove those types :smile: